apps/mark-scan/frontend/src/pages/ballot_invalidated_page.tsx

```diff
@@ -8,13 +8,19 @@ interface Props {
   authStatus:
     | InsertedSmartCardAuth.CardlessVoterLoggedIn
     | InsertedSmartCardAuth.PollWorkerLoggedIn;
+  paperPresent: boolean;
 }
 
-export function BallotInvalidatedPage({ authStatus }: Props): JSX.Element {
+export function BallotInvalidatedPage({
+  authStatus,
+  paperPresent,
+}: Props): JSX.Element {
   return (
     <AskPollWorkerPage
       authStatus={authStatus}
-      pollWorkerPage={<RemoveInvalidatedBallotPage />}
+      pollWorkerPage={
+        <RemoveInvalidatedBallotPage paperPresent={paperPresent} />
```

Comment:
I noticed in the video that the "Ask a Poll Worker for Help" text wraps awkwardly to two lines. I think we should look into moving the left-aligned text. Also, the "Voting Session in Progress" screen could use some more horizontal padding - it's butting right up against the sides. But these are minor issues that can be addressed in a future UI polish pass.

Reply:
The "Ask a Poll Worker for Help" screen uses the `CenteredPageLayout` component, which is used by a lot of the app. Agree we should do a design pass on that component and make sure we update all callers at once for consistency.

"Voting Session in Progress" padding was a one-off fix though:

Before

![Screenshot 2024-01-09 at 11 48 21 AM](https://github.com/votingworks/vxsuite/assets/1060688/ca3dae42-07df-4b8c-b280-5e9b5c4eeec2)


After

![Screenshot 2024-01-09 at 11 48 11 AM](https://github.com/votingworks/vxsuite/assets/1060688/3c51dc82-bb08-477a-be6b-a250c14005fd)

It has the same wrapping awkwardness with padding, but at least it's consistent so IMO it's the lesser of two evils. We can fix this and other screens in https://github.com/orgs/votingworks/projects/23/views/75?pane=issue&itemId=49471078